segment_battery.go

```diff
@@ -30,10 +30,18 @@ const (
 	ChargingColor Property = "charging_color"
 	// DischargingColor to display when discharging
 	DischargingColor Property = "discharging_color"
+	// HideWhileCharging Hide the battery icon while it's charging
+	HideWhileCharging Property = "hide_while_charging"
```

Comment:
I would rename this to `display_charging` to stay consistent with the other switches.